LEAF_Request_Portal/sources/Form.php

```diff
@@ -2120,32 +2090,28 @@ public function isMasked($indicatorID, $recordID = null)
      */
     public function isNeedToKnow($recordID = null)
```

Comment:
And here.  Just take a look at these methods to see if they can be cleaned up a bit without adding too much extra work on things.